src/domain/synthetics/tradingview/SyntheticsTVDataProvider.ts

```diff
@@ -19,7 +19,7 @@ export class SyntheticsTVDataProvider extends TVDataProvider {
     period: string,
     onFallback?: (ex: Error) => void
   ): Promise<FromOldToNewArray<Bar>> {
-    const limit = 5000;
+    const limit = 500;
```

Comment:
Seems that for 1m period this will fetch bars for last ~8 hours. Which doesn't cover full X-axis width, not sure if it's ok. As for me it's ok, as it doesn't make much sense to zoom out more than 12hours range with 1m period, so if it's intended just ignore this commit.